docs/sphinx/developers/units.txt

```diff
@@ -0,0 +1,52 @@
+Units of measurement
+====================
+
+Since Bio-Formats 5.1 and the adoption of the 2015-01 OME Data Model,
+the data model and the corresponding Bio-Formats model and metadata
+APIs have added support for units of measurement.  Previously, the
+units for various properties such as the physical size of an image,
+stage position, confocal pinhole size, light wavelengths etc. were
+fixed in the model.  This was however somewhat inflexible, and not
+appropriate for imaging modalities at widely different scales, and the
+solution to this was to add a unit of measurement to each of these
+properties.  The image size, for example, was previously specified to
```

Comment:
I would probably break this into 2 sentences :
This was however somewhat inflexible, and not appropriate for imaging modalities at widely different scales. The solution to this was to add a unit of measurement to each of these properties.